loky/backend/context.py

```diff
@@ -52,13 +52,20 @@ def assert_spawning(obj):
             )
 
     def get_context(method="loky"):
-        return LokyContext()
+        if method == "loky":
+            return LokyContext()
+        elif method == "loky_no_main":
+            if sys.platform == "win32":
+                return LokyContext()
```

Comment:
The loky process that we implemented is only working for Posix. For windows, we are using the stock `Process` and I cannot modify it. We could clone the code in `loky` for windows but it is more maintenance. I am not against it but we should discuss it fully.